use-timescale/continuous-aggregates/about-continuous-aggregates.md

```diff
@@ -55,13 +55,51 @@ these restrictions:
     clause does not matter.
 *   You must use an `INNER JOIN`, no other join type is supported.
 *   The `JOIN` conditions can only be equality conditions.
-*   The hypertable on the `JOIN` condition must be a hypertable, and not a continuous
-     aggregate. Additionally, you can't use joins in hierarchical continuous aggregates.
+*   The hypertable on the `JOIN` condition can be a hypertable, and not a continuous
+    aggregate. Additionally, you can't use joins in hierarchical continuous aggregates.
```

Comment:
```suggestion
*   The table on the `JOIN` condition can be a hypertable, it does not have to be a continuous
     aggregate. However, you can't use joins in hierarchical continuous aggregates.
```

Reply:
It cannot be a continuous aggregate, but all of this wording is confusing.